### DISCUSSION TITLE
"No existing project" when trying to create postgres

### DISCUSSION THREAD

**Discussion by rlimberger** [ubicloud/ubicloud#1996] [General]
I am trying to create a Postgres resource on my Ubicloud cloudyfied Hetzner server and am getting "No existing project" error despite there being a project. What could cause this?

```
"GET /project/pjtmr22rmrsj0nsdzbxnqbytq9/postgres/create HTTP/1.1" 200 57527 0.0446
ubicloud-app          | 16:55:28 web.1       | RuntimeError: No existing project
ubicloud-app          | 16:55:28 web.1       | /app/prog/vnet/subnet_nexus.rb:9:in `assemble'
```

**Comment by enescakir** [comment]
You need a project to provision underlying resources for Postgres, like a virtual machine and a private subnet. You can create a project and set its ID to the `POSTGRES_SERVICE_PROJECT_ID` variable.

https://github.com/ubicloud/ubicloud/blob/cfd27208cb51c3c989e3b9760593cb402b53d1da/config.rb#L132

https://github.com/ubicloud/ubicloud/blob/cfd27208cb51c3c989e3b9760593cb402b53d1da/prog/postgres/postgres_resource_nexus.rb#L57

**Comment by rlimberger** [comment]
I have created a project and the error im seeing happens when trying to setup a postgres resource within that project. 

<img width="1262" alt="Screenshot 2024-09-30 at 10 44 05 AM" src="https://github.com/user-attachments/assets/6bb7e42e-98d0-4341-a2bc-638bd3bd0775">
